app/views/item/AddAdditionalInformationYesNoView.scala.html

```diff
@@ -45,7 +45,8 @@
                 heading = messages("item.addAdditionalInformationYesNo.heading"),
                 caption = Some(messages("item.additionalInformation.caption", itemIndex.display)),
                 additionalHtml = messages("item.addAdditionalInformationYesNo.paragraph").toParagraph
-            )
+            ),
+            hint = Some(HtmlContent(messages("item.addAdditionalInformationYesNo.hint")))
```

Comment:
`HtmlContent` is designed for raw html. Use `Text` as the content type when the input is a string

Reply:
I followed suit with an existing hint in the repo, should I change those too?

Reply:
Yes please 👍